### DISCUSSION TITLE
Hide/remove "take a screenshot" option?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [Floorp-Projects/Floorp#498] [Feature Request]
Hello,

Your tool to take screenshot is good but not enough.
I have a tool to take screenshot on my linux + I use this thing to take long page screenshots https://addons.mozilla.org/fr/firefox/addon/handy-screenshot/
this thing allows to quickly edit with useful tools, and this makes it more useful
Therefore how can I remove it from the right click menu?

(Or you could improve the screenshot tool haha)

**Comment by Ryosuke-Asano** [comment]
https://wiki.mozilla.org/Firefox/Screenshots/FAQs

**Reply by BarbzYHOOL** [reply]
oh okay, it's a firefox new feature? 
thanks
